eng/mgmt/automation/generate_utils.py

```diff
@@ -77,7 +78,9 @@ def generate(
         )
     logging.info(command)
     if os.system(command) != 0:
-        logging.error('[GENERATE] Autorest fail')
+        error_message = '[GENERATE][Error] Autorest fail'
```

Comment:
Instead of using "Autorest fail" we should use "code generation failed" and include any other details or links to documentation in the error message (at least an email or Teams channel link to reach out).

Reply:
One problem (particular for Autorest) is that without reading the output of the automation, it be hard (even for me) to know by the look, whether it is caused by Swagger problem or codegen problem.
https://github.com/Azure/azure-sdk-for-java/pull/39953#discussion_r1582680718

I believe there is near to no lib on DPG that still does Swagger automation (the full codegen pass -- not include the handwritten vanilla path).

For mgmt, we had daily run to make sure we know what could have problem https://github.com/Azure/autorest.java/wiki/Fluent-Codegen-Report

---

For failure on TypeSpec emitter, I will add link to say something like "report bug on autorest.java https://github.com/Azure/autorest.java/issues"